src/components/DetailsPane/DetailsPane.scss

```diff
@@ -0,0 +1,38 @@
+@use "../../styles/variables.scss" as *;
+.detailsPane{
+position: fixed;
+top: 0;
+left: 0;
+width: 100%;
+height: 100%;
+display: flex;
+align-items: center;
+justify-content: center;
+background-color: rgba(0, 128, 0, 0);
+}
+.detailsPane__Container{
+    width: 96vw;
+    height: 96vh;
+    background-color: rgb(255, 255, 255);
+    border: solid 3px black;
+    border-radius: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+}
+.detailsPane__Header{
+    display: flex;
+}
+.button--Close{
+    border: solid rgba(255, 0, 0, 0) 3px;
+    background-color: rgba(255, 0, 0, 0);
+    width: 3rem;
+}
+.detailsPane__SaveBtnContainer{
+    display: flex;
+    justify-content: center;
+}
+.button--Save{
+    background-color: $orange-color-palette;
+    border-radius: 1.5rem;
+}
```

Comment:
this class should be on button.scss and lowercase hypenated too: `.button--save`